### PR TITLE
Update to add the camel-servicenow extension

### PR DESCRIPTION
**Pull request by zhfeng**
Fix https://github.com/apache/camel-quarkus/issues/766

### CODE REVIEW

File: extensions/servicenow/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+metadata:
+  keywords:
+  - "camel"
+  - "api"
+  - "cloud"
+  - "management"
+  guide: "https://quarkus.io/guides/camel"
+  categories:
+  - "integration"
+  status: "stable"
```
[the remark this answers is not in the document]
Comment (zhfeng) [reply extensions/servicenow/runtime/src/main/resources/META-INF/quarkus-extension.yaml]: I agree.

File: integration-tests/servicenow/pom.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        <profile>
+            <id>native</id>
+            <activation>
+                <property>
+                    <name>native</name>
+                </property>
+            </activation>
+            <properties>
+                <quarkus.package.type>native</quarkus.package.type>
+		<quarkus.native.additional-build-args>-H:EnableURLProtocols=https</quarkus.native.additional-build-args>
```
[the remark this answers is not in the document]
Comment (zhfeng) [reply integration-tests/servicenow/pom.xml]: Thanks for pointing it out !

**Comment by zhfeng** [comment]
In term of the "Dependency convergence" errors with jdk11 and jdk12
```
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (camel-quarkus-enforcer-rules) @ camel-quarkus-servicenow ---
[WARNING] 
Dependency convergence error for javax.xml.soap:javax.xml.soap-api:1.4.0 paths to dependency are:
+-org.apache.camel.quarkus:camel-quarkus-servicenow:1.1.0-SNAPSHOT
  +-org.apache.camel:camel-servicenow:3.1.0
    +-org.apache.cxf:cxf-core:3.3.5
      +-javax.xml.ws:jaxws-api:2.3.1
        +-javax.xml.soap:javax.xml.soap-api:1.4.0
and
+-org.apache.camel.quarkus:camel-quarkus-servicenow:1.1.0-SNAPSHOT
  +-org.apache.camel:camel-servicenow:3.1.0
    +-org.apache.cxf:cxf-core:3.3.5
      +-com.sun.xml.messaging.saaj:saaj-impl:1.4.0-b03
        +-javax.xml.soap:javax.xml.soap-api:1.4.0-b01

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability the error(s) are [
Dependency convergence error for javax.xml.soap:javax.xml.soap-api:1.4.0 paths to dependency are:
+-org.apache.camel.quarkus:camel-quarkus-servicenow:1.1.0-SNAPSHOT
  +-org.apache.camel:camel-servicenow:3.1.0
    +-org.apache.cxf:cxf-core:3.3.5
      +-javax.xml.ws:jaxws-api:2.3.1
        +-javax.xml.soap:javax.xml.soap-api:1.4.0
and
+-org.apache.camel.quarkus:camel-quarkus-servicenow:1.1.0-SNAPSHOT
  +-org.apache.camel:camel-servicenow:3.1.0
    +-org.apache.cxf:cxf-core:3.3.5
      +-com.sun.xml.messaging.saaj:saaj-impl:1.4.0-b03
        +-javax.xml.soap:javax.xml.soap-api:1.4.0-b01
```
It looks like an issue with the cxf 3.3.5 and had been fixed in the upstream when replacing to use the [jakarta api](https://github.com/apache/cxf/commit/6fabd55129e5bf6c0768a61aa649411dfdd98ddd#diff-04dc5d029560773ac79faaf3da0fbb22) in [CXF-7910 Change JAX-WS javax to jakarta artifact dependencies](https://issues.apache.org/jira/browse/CXF-7910) which will be included in the cxf 3.4.0 release plan.
Since the camel-servicenow does not use any ws transport, it might be safe to exclude these artifacts ?

**Comment by ppalaga** [comment]
> it might be safe to exclude these artifacts ?

Probably. Try it and if your tests pass, we are likely good to go.

**Pull request merged**